### PR TITLE
Feature/baip 1564 dms file upload gql

### PR DESCRIPTION
**Pull request by bennyvdhoogen**

### CODE REVIEW

File: src/schema/document/document.resolver.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import { Args, Context, Mutation, Parent, Query, ResolveField, Resolver } from '@nestjs/graphql';
```
Comment (fortunato) [inline src/schema/document/document.resolver.ts:1]: Why no CQRS?

File: src/schema/document/document.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		@Args('fileName') fileName: string,
+		@Args({ name: 'provider', defaultValue: 'dms' }) provider: string,
+		@Context() ctx: any,
+	): Promise<DmsUploadUrlResponse> {
+		return this.documentService.getDocumentUploadUrl(assetCode, fileName, provider, ctx);
+	}
+
+	@Query(() => [DMSDocumentSpanInstallationGQLModel])
+	@Roles({ roles: ['realm:aip_owner', 'realm:aip_admin', 'realm:aip_survey'], mode: RoleMatchingMode.ANY })
+	public async getDocumentsSpanInstallation(
```
Comment (fortunato) [inline src/schema/document/document.resolver.ts:30]: Should this be `findDocumentsSpanInstallation`? Or `findSpanInstallationDocuments` for a grammarly better option?

File: src/schema/survey/types/InspectionStandard.ts [hunk 3]
```diff
@@ -12,4 +14,29 @@ export enum InspectionStandard {
 	quaywalls = 'quaywalls',
 	verificatieBerekening = 'verificatieBerekening',
 	spanInstallation = 'spanInstallation',
+	facadeFollowUpSurvey = 'facadeFollowUpSurvey',
+	spanMeasuresSurvey = 'spanMeasuresSurvey',
+}
+
+export function getSurveyInspectionType(inspectionStandardType: Survey['inspectionStandardType']): InspectionStandard {
```
Comment (fortunato) [inline src/schema/survey/types/InspectionStandard.ts:21]: Why do we need a switch if we have a locally declared enum? All keys seem to match?

File: src/schema/document/models/document.model.spec.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { Document } from './document.model';
```
[the remark this answers is not in the document]
Comment (bennyvdhoogen) [reply src/schema/document/models/document.model.spec.ts:1]: It is! Was testing out Copilot's ability to automatically generate files. I always have a hard time determining what to test in a test of a model file; but you're right there's not need to initialise the doc individually.

File: src/schema/document/document.resolver.ts (hunk 2 above)
Comment (bennyvdhoogen) [reply src/schema/document/document.resolver.ts:30]: Good one. I prefer the first one over the latter as it ensures that all following queries for other inspection types start with `findDocuments...`

File: src/schema/document/document.resolver.ts (hunk 1 above)
Comment (bennyvdhoogen) [reply src/schema/document/document.resolver.ts:1]: Good one. Will change.

**Pull request merged**